### DISCUSSION TITLE
Screenshoting doesn't seem to work :(

### DISCUSSION THREAD

**Discussion by unt3rhofer** [i3/i3#6143] [Q&A] [answered]
Hello! I just recently set up i3 on my laptop (Fedora release 40) and can't get screenhoting to work. I don't really care about fancy selection screenshoting or anything similar, I would just like good'ol PrtSc to work and it doesen't :(.
What I mean by "doesen't work": 
-I press PrtSc
-can't find the screenshot in any folder
-screenshot is not in the clipboard

Hope you guys can help!
Thanks!

This is what I tried to do but didn't have any luck:
bindsym --release Print exec NOW=$(date +%d-%b-%Y_%H-%M-%S) && maim --format png --select > ~/screenshots/screenshot_$NOW.png && xclip -selection clip -t image/png ~/screenshots/screenshot_$NOW.png
source for this attempt: https://gist.github.com/Dechrissen/db67f2a79a4cdd00909d679adefe41d0

Here is my i3 config file:
[config.zip](https://github.com/user-attachments/files/16133924/config.zip)

**Comment by orestisfl** [comment]
Try
```
bindsym --release Print exec "NOW=$(date +%d-%b-%Y_%H-%M-%S) && maim --format png --select > ~/screenshots/screenshot_$NOW.png && xclip -selection clip -t image/png ~/screenshots/screenshot_$NOW.png"
```

**Reply by unt3rhofer** [reply]
I am a bit dumb and I left the "~" in the path so poor PrtSc could not find where to save the screenshots.

**Reply by unt3rhofer** [reply, marked as answer]
Alright, yesterday I was happy with this result because it actually made a file. The problem is, the file had 0kB which I didn't catch yesterday when marking this as the aswer. What eventually worked for me was dowloading "scrot" and using it this way:
scrot -e 'mv $f /home/martin/Pictures/Screenshots'
Now you can just bindsym it to some shortcut of your choice usic exec.
